doc/src/build/move/build-test.md

```diff
@@ -266,34 +266,45 @@ function.
     fun test_sword_transactions() {
         use sui::test_scenario;
 
-        let admin = @0xABBA;
+        // create test addresses representing users
+        let admin = @0xBABE;
         let initial_owner = @0xCAFE;
         let final_owner = @0xFACE;
 
-        // first transaction executed by admin
-        let scenario = &mut test_scenario::begin(&admin);
+        // first transaction to emulate module initialization
+        let scenario_val = test_scenario::begin(admin);
+        let scenario = &mut scenario_val;
         {
+            init(test_scenario::ctx(scenario));
+        };
+        // second transaction executed by admin to create the sword
+        test_scenario::next_tx(scenario, admin);
+        {
+            let forge = test_scenario::take_from_sender<Forge>(scenario);
             // create the sword and transfer it to the initial owner
-            sword_create(42, 7, initial_owner, test_scenario::ctx(scenario));
+            sword_create(&mut forge, 42, 7, initial_owner, test_scenario::ctx(scenario));
+            test_scenario::return_to_sender(scenario, forge)

```

Comment:
`sword_create()` doesn't take a `Forge` as a parameter. I think all that is needed here is:

```
sword_create(42, 7, initial_owner, test_scenario::ctx(scenario));
```